### DISCUSSION TITLE
Which version of C# required for translated code?

### DISCUSSION THREAD

**Discussion by ghost** [pfusik/cito#102] [Q&A] [answered]
This is a very important detail that should be available on the project's README.md.

**Comment by pfusik** [comment, marked as answer]
Again this depends on what you transpile. Some code is compatible with .NET 1.0. I think there are some features emitted for .NET 5, but not any later versions.

**Reply by ghost** [reply]
Which features? I think I need to know because I haven't yet jumped to .NET Core.

**Reply by pfusik** [reply]
.NET Framework doesn't support:
- `foreach ((key, value) in dictionary)`
- `Math.Cbrt`
